yarn.lock

```diff
@@ -1328,20 +1328,23 @@
     "@types/istanbul-reports" "^1.1.1"
     "@types/yargs" "^12.0.9"
 
-"@kiwicom/account-native@^0.0.1":
-  version "0.0.1"
-  resolved "https://registry.yarnpkg.com/@kiwicom/account-native/-/account-native-0.0.1.tgz#f9d9a0504e4d4373cf1cdf5b402b5e1ab375ebaa"
-  integrity sha512-Ggu+1g1tS8hLOloUD+Bt9SdrpiybWmDUJ+k7mWvlb1emfEcRmob3+ANbSA/uXRiHdx204OcSmtEEzZXWX+QQaA==
+"@kiwicom/account-native@^0.0.4":
+  version "0.0.4"
+  resolved "https://registry.yarnpkg.com/@kiwicom/account-native/-/account-native-0.0.4.tgz#300411e7d29ebb85e20bf8dbffbf9581df788f9b"
+  integrity sha512-N03UKZR6+CLQTZX9s8/JE3Six1vtxqR07TtUO4wSRxrp/Ehq84sZ3te4FF2bEdglpJnmCXfgX+jhFZXy21uNoA==
   dependencies:
     "@kiwicom/graphql-global-id" "^0.10.0"
     "@kiwicom/graphql-utils" "^0.8.0"
+    "@kiwicom/js" "^0.13.0"
+    "@kiwicom/orbit-components" "^0.53.0"
     "@kiwicom/orbit-design-tokens" "^0.6.5"
     "@kiwicom/relay" "^3.2.0"
     buffer "^5.2.1"
     dataloader "^1.4.0"
```

Comment:
Why do you need `dataloader`? That should be useful for the GraphQL server part, not in the client, right?

Reply:
Account don't have a backend yet, so it uses an in-memory-grahpql-mock, it is needed for now

Reply:
Actually, I think it is in account just because someone copy-pasted it from another package. I will remove it for next release as well.